src/redux/clockifyThunk.ts

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import { AUTH_TOKEN, WORKSPACE_ID } from "../config";
+import { 
+    Data, 
+    TimeEntryDataProp, 
+} from "../types/types";
+
+export const createTimeEntry = createAsyncThunk("createTimeEntry", async (timeEntryData: TimeEntryDataProp) => {
+    try{
+        const response = await fetch(`https://api.clockify.me/api/v1/workspaces/${WORKSPACE_ID}/time-entries`, {
+            method: "POST",
+            headers: {
+                'X-Api-Key':AUTH_TOKEN ?? '',
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(timeEntryData)
+        })
+    
+        if (!response.ok) {
+            throw new Error('Failed to create time entry')
```

Comment:
it needs to log the actual error to make sense out of it if this ever happens rit?

Reply:
https://github.com/PreethiKalyani99/Clockify-Typescript/commit/6020255fafe03fd56faf35ef72041c48daa8cc9e